### PR TITLE
release v1.15.0

### PR DESCRIPTION
**Pull request by benmosher**
Should've really just done this last week 😳

- [x] once-over from @jfmengels 
- [ ] tag
- [ ] publish
- [ ] merge package.json back to master

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -289,9 +299,19 @@ for info on changes for earlier releases.
 [`no-mutable-exports`]: ./docs/rules/no-mutable-exports.md
 [`prefer-default-export`]: ./docs/rules/prefer-default-export.md
 [`no-restricted-paths`]: ./docs/rules/no-restricted-paths.md
-
+<<<<<<< HEAD
+[`no-absolute-path`]: ./docs/rules/no-absolute-path.md
+=======
+[`max-dependencies`]: ./docs/rules/max-dependencies.md
+>>>>>>> tizmagik/feature/max-dependencies
```
Comment (jfmengels) [inline CHANGELOG.md]: Ahah! My publish reviews are finally useful: There's are traces of merge conflicts here :smile:

Comment (benmosher) [inline CHANGELOG.md]: 😅 I need a pre-commit hook for this. if even just for the changelog.

Comment (jfmengels) [inline CHANGELOG.md]: Yeah, there should be pre-commit hooks that warn when there are strings like `<<<<<<<` anywhere in your project. Maybe an idea for https://github.com/SamVerschueren/clinton @SamVerschueren?

**Comment by jfmengels** [comment]
LGTM :)

I'm fine with releasing as is, but we're so close to merging https://github.com/benmosher/eslint-plugin-import/pull/485 that it would be a shame not to include it right away. We could merge it and fix that last comment ourselves. What do you think?

**Comment by benmosher** [comment]
I'm cool with that, wasn't sure how fast it'd move today and I wanted to get the rest out the door.

If you can merge it into master and rebase this branch back onto the new master, I'm up for it? or I can knock it out in the morning tomorrow probably

**Comment by benmosher** [comment]
Plus #555 needs review + will probably merge soon, so 1.16 is probably days away anyway. so I'm cool either way

**Comment by jfmengels** [comment]
I'd like to concentrate on writing a blog post (my first one :o) tonight, so I'll leave that to you if you don't mind (plus, I'm not sure I got the whole `master`/`new master` branch thing :sweat_smile:).
And ok, if #555 gets merged in a few days, I'm fine with postponing #485 too.

**Comment by benmosher** [comment]
sweet deal, no worries. I'll probably punt on #485 until v1.16, and then release that with #555. will need to wait until tomorrow either way.

some part of me still wants releases to be substantial, even though I'm on the CI/CD train. old habits, I guess. 😄 

blog post sounds way more important/exciting, get after it! 😎

File: CHANGELOG.md (hunk 1 above)
Comment (SamVerschueren) [inline CHANGELOG.md]: Very good suggestion @jfmengels! Will add it.

**Pull request merged**